django_check_seo/cms_toolbars.py

```diff
@@ -3,7 +3,7 @@
 # Third party
 from cms.toolbar_base import CMSToolbar
 from cms.toolbar_pool import toolbar_pool
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_lazy as _
```

Comment:
Django check seo should still be compatible with old python2/django1.11 projects (yes they still exist in the wild), and with python3/dango2.2 Can you add a try/except like this one instead of breaking the compatibility of old versions?
```py
try:
    from django.utils.translation import ugettext_lazy as _
except:
    from django.utils.translation import gettext_lazy as _
```